sass/mixins/box/box.scss

```diff
@@ -0,0 +1,24 @@
+@import "palette.scss";
+@import "dimensions.scss";
+@import "typography.scss";
+
+@mixin box($type) {
+
+  box-sizing: border-box;
+  display: block;
+  padding: 10px;
+
+  @if $type == 'primary' {
+    background-color: white;
+  }
+  @else if $type == 'secondary' {
+    background-color: $color-150-grey;
+  }
+  @else if $type == 'tertiary' {
+    background-color: $color-300-grey;
+  }
+
+  &[skin=large] {
+    padding: 7px 15px;
```

Comment:
Are they entirely specific though? At the moment in Composer, sizes are based on a "base spacing unit". I suspect we do want to try to maintain a relationship between different sizes in the application.